composer.json

```
@@ -10,7 +10,7 @@
         }
     ],
     "conflict": {
-        "doctrine/dbal": "2.9.* <2.9.3 || 3.3.0",
+        "doctrine/dbal": "2.9.* <2.9.3 || 3.3.0 || 3.5.0",
```

```suggestion
        "doctrine/dbal": "2.9.* <2.9.3 || 3.3.0 || 3.5.*",
```
Maybe even conflict with 3.5.* for now as this does not seem to get fixed https://github.com/doctrine/dbal/pull/5779#issuecomment-1288159168

---

IMHO yes, releases were quite frequent recently and I'm not sure we can otherwise find a workaround in time.